Java-practicals/Exp 4/Array of objects/bank.java

```diff
@@ -3,15 +3,16 @@
     int topPointer;
     bank(int numberOfAccounts){
         arrayOfAccounts = new account[numberOfAccounts];
-        topPointer=0;
+        topPointer=-1;
```

Comment:
set topPointer value to point the last element and not the place to be added next